### DISCUSSION TITLE
Can a Security Onion sensor ingest AWS VPC flow logs written to S3?

### DISCUSSION THREAD

**Discussion by ihuxn4hi** [Security-Onion-Solutions/securityonion#10048] [Unsupported Versions]
Is it possible for a Sensor (or Sensors) to ingest and analyze network traffic from a VPC flow log that's in S3?

Thanks!

**Comment by InfosecGoon** [comment]
You should be able to ingest those logs using the vpcflow option in the AWS filebeat module.

More information: https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-aws.html
